src/elements/Button/Button.styl

```diff
@@ -123,17 +134,19 @@
   transition-duration $duration-shortest
 
   // Children
-
   & > .content
-    padding $space-normal
-
+    padding $space-zero
+    padding-bottom 7px
     &.has-addons
       border-left 1px solid var(--button-border-color)
-
+      & > .addons
+         &.padded
+            padding $space-normal	
+  // States	  // States
   & > .addons
     &.padded
       padding $space-normal
-
+      
```

Comment:
Remove trailing spaces across all files. Ideally you should configure your IDE or editor to do this automatically on save.